redux/book/book.js

```diff
@@ -0,0 +1,30 @@
+// ACTIONS
+const REMOVE__BOOK = 'bookstore/src/redux/book/REMOVE__BOOK';
+const ADD__BOOK = 'bookstore/src/redux/book/ADD__BOOK';
```

Comment:
- [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions like:
```javascript
<app name>/<reducer name>/<action type>
```
for example:
```javascript
const REMOVE_BOOK = 'book-store-app/book/REMOVE_BOOK';
```